src/glossary.rst

```diff
@@ -412,11 +412,20 @@ async block expression
 :dp:`fls_p6nvfs7bfoxd`
 An :dt:`async block expression` is a :t:`block expression` that is specified
 with :t:`keyword` ``async`` and encapsulates behavior which is executed in
-asynchronous manner.
+an asynchronous manner.
 
 :dp:`fls_je689rormhd6`
 See :s:`AsyncBlockExpression`.
 
+.. _fls_lYrTaCM1LcXU:
+
+control flow boundary
+^^^^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_EXoGOkCRsfKK`
+An :dt:`async control flow boundary` is a :t:`control flow boundary` that
+additionally allows suspending the execution via :t:`[await expression]s`.
```

Comment:
How about

An async control flow boundary is a control flow boundary that allows the suspension of execution via await expressions.